### ISSUE TITLE
extension translation

### ISSUE THREAD

**Issue by datalocale** [ckan/ckanext-issues#66]
Hi,
I do not see in the code a mecanism for string translation. Do you plan on adding it ? Can we contribute to the project by adding this possibility ? In that cas what would be your prefered system ?

**Comment by davidread** [comment]
You're right, we don't have any translation in this. I'd be more than happy for this to be added, following ckan and ckanext conventions. I'm no expert on this.

**Comment by datalocale** [comment]
ok We'll do that. Thanks
